kvbc/tools/db_editor/include/kv_blockchain_db_editor.hpp

```diff
@@ -301,6 +301,43 @@ struct GetCategories {
   }
 };
 
+struct GetLatestCategoryInfo {
+  const bool read_only = true;
+  std::string description() const {
+    return "getLatestCategoryInfo CATEGORY-ID [BLOCK-VERSION]\n"
+           "  Returns the latest blockID that contains the givenb category, started from BLOCK-VERSION. If "
+           "BLOCK-VERSION is not set, we start from the lastReachableBlock";
+  }
+
+  std::string execute(const KeyValueBlockchain &adapter, const CommandArguments &args) const {
+    if (args.values.empty()) {
+      throw NotFoundException{"No Category ID was given"};
+    }
+    auto latestBlockID = adapter.getLastReachableBlockId();
+    if (args.values.size() == 2) {
+      latestBlockID = toBlockId(args.values[1]);
+    }
+    auto cat = args.values.front();
+    std::map<std::string, std::string> cat_updates_map;
+    BlockId relevantBlockId = adapter.getGenesisBlockId();
+    for (auto block = adapter.getGenesisBlockId(); block <= latestBlockID; block++) {
```

Comment:
Actually, I might have gotten the command wrong. Do we want to always search from genesis to some block ID? Or we want to search from some block ID to the end?

Reply:
done